docs/index.rst

```diff
@@ -44,7 +44,7 @@ The histolab package can be installed by using::
 
 Prerequisites
 *************
-Please see `installation instructions <https://github.com/histolab/histolab/blob/master/docs/installation.rst>`_.
+Please see `installation instructions <installation.html>`_.
```

Comment:
this link doesn't exsist 🤷🏽

Reply:
what do you mean? if you check the docs build it works